cmd/upgrade/upgrade.go

```
@@ -0,0 +1,380 @@
+	}
+
+	return &release, nil
+}
+
+// findAsset 查找匹配当前 OS/Arch 的 release asset
+func findAsset(release *releaseInfo) (*asset, error) {
+	goos := runtime.GOOS
+	goarch := runtime.GOARCH
+
+	// 标准化 arch 名称以匹配 release asset 命名
+	archName := goarch
```

golangci-lint (2.12.2)

[error] ineffectual assignment to archName

(ineffassign)